frontend/src/scenes/surveys/surveyViewViz.tsx

```diff
@@ -92,12 +102,12 @@ export function UsersStackedBar({ surveyUserStats }: { surveyUserStats: SurveyUs
                     <div className="w-full flex justify-center">
                         <div className="flex items-center">
                             {[
-                                { count: seen, label: 'Viewed', color: 'bg-primary' },
-                                { count: dismissed, label: 'Dismissed', color: 'bg-warning' },
-                                { count: sent, label: 'Submitted', color: 'bg-success' },
-                            ].map(({ count, label, color }) => (
+                                { count: seen, label: 'Viewed', style: { backgroundColor: '#1D4AFF' } },
```

Comment:
curious why this change? I think we're moving away from specific styles on components to utility classes for those styles (but not 100% sure)

Reply:
The colors suggested by Cory (https://github.com/PostHog/posthog/pull/17815) are different from those available in Tailwind.

Probably worth a separate PR to make them available there.